templates/README.md

```diff
@@ -0,0 +1,54 @@
+# Using a template to add a new page into the project
+
+All of this should be done without the UI Designer running. <br/>
+Everything in-between the `%` signs should be replaced. 
+For all of these, keep the same naming conventions as the templates.
+
+## Adding a new page/form/layout
+
+Copy the `%template-page-name%/` directory into the `uid-pages/` folder.
+
```

Comment:
Need to add a comment to remind that we need to add this page in the "settings.gradle"

Reply:
without this, it is perfect.